src/pages/library/dataBrowser-utils.js

```diff
@@ -115,11 +128,9 @@ export const useDataCatalog = () => {
     withErrorReporting('Error loading data catalog'),
     Utils.withBusyState(setLoading)
   )(async () => {
-    const datasets = await Ajax(signal).Catalog.getDatasets()
-    const normList = _.map(dataset => {
-      const normalizedDataset = normalizeDataset(dataset)
-      return _.set(['tags'], extractTags(normalizedDataset), normalizedDataset)
-    }, datasets.result || [])
+    const { result: datasets } = await Ajax(signal).Catalog.getDatasets()
+    const dataCollectionsToInclude = getEnabledBrand().catalogDataCollectionsToInclude
+    const normList = filterAndNormalizeDatasets(datasets, dataCollectionsToInclude)
```

Comment:
How are we handling the case where `getEnabledBrand()` returns something that doesn't have the `catalogDataCollectionsToInclude` property defined?

Reply:
(This line here: https://github.com/DataBiosphere/terra-ui/pull/3441/files#diff-2f8165b464e0d758e22826eef8df6b69ed77b32fd302aeeecc056801a29a8540R112